asapdiscovery-alchemy/asapdiscovery/alchemy/tests/test_cli.py

```diff
@@ -129,18 +129,24 @@ def test_alchemy_prep_run_with_fails(tmpdir, mac1_complex):
     with tmpdir.as_cwd():
         # complex to a local file
         mac1_complex.to_json_file("complex.json")
+        # write out the workflow to file
+        openeye_prep_workflow.to_file("openeye_workflow.json")
 
         result = runner.invoke(
             alchemy,
             [
                 "prep",
                 "run",
+                "-f",
+                "openeye_workflow.json",
                 "-n",
                 "mac1-testing",
                 "-l",
                 ligand_file.as_posix(),
                 "-r",
                 "complex.json",
+                "-p",
+                1,
```

Comment:
is the test system single core? or can we speed up the test by setting to `auto`?